src/main/java/Entities/Board.java

```diff
@@ -63,7 +71,8 @@ public void removePlayer(Player player){
      * @return This returns a Cell which the Player instance is on.
      */
     public Cell getPlayerCell(Player player){
-        return this.cells.get(this.playerPositions.get(player));
+        int position = player.getPosition();
+        return this.getCell(position);
```

Comment:
Great change. I don't think it changes anything in terms of functionality but its a lot more readable.